### PR TITLE
[14.0][MIG] Migrate module: base_search_fuzzy to 14.0

### PR DESCRIPTION
**Pull request by AreebSiddiqiIR**

### CODE REVIEW

**Comment by AreebSiddiqiIR** [comment]
@moylop260

**Comment by moylop260** [comment]
Hi @AreebSiddiqiIR 
The branch has conflicts with the base

Could you rebase with last commit and fix conflicts, please?

**Comment by AreebSiddiqiIR** [comment]
> Hi @AreebSiddiqiIR
> The branch has conflicts with the base
> 
> Could you rebase with last commit and fix conflicts, please?

I am not sure where to find the conflicts. All the modified files are inside the directory "/base_search_fuzzy".  Please guide.

**Comment by moylop260** [comment]
Forge it
It looks fine!

File: base_search_fuzzy/models/trgm_index.py [hunk 1]
```diff
@@ -41,6 +42,7 @@ class TrgmIndex(models.Model):
         string="Index Type",
         default="gin",
         required=True,
+        ondelete={"gin": "set default", "gist": "set default"},
         help='Cite from PostgreSQL documentation: "As a rule of thumb, a '
```
Comment (gdgellatly) [inline base_search_fuzzy/models/trgm_index.py]: This help message is a bit old, pg9.6 old. The performance tradeoff in maintaining GIN indexes in modern postgres is negligible. In fact it is removed from documentation. Maybe we are better to just reference the docs here.

https://www.postgresql.org/docs/current/textsearch-indexes.html

**Pull request closed without merging**